testdata/engine/loader/reviewpad_after_action_transform.yml

```
@@ -0,0 +1,19 @@
+# Copyright 2022 Explore.dev Unipessoal Lda. All Rights Reserved.
+# Use of this source code is governed by a license that can be
+# found in the LICENSE file.
+
+api-version: reviewpad.com/v1alpha
+
+rules:
+  - name: tautology
+    kind: patch
+    description: Always true
+    spec: true
+
+workflows:
+  - name: test-process
+    description: Test process
+    if:
+      - rule: tautology
+    then:
+      - '$merge("merge")'
```

I understand the need of theses files and I'm all for it. My suggestion is to reduce the file content to only what it's actually needed.

```suggestion
api-version: reviewpad.com/v3.x

rules:
  - name: tautology
    kind: patch
    spec: true

workflows:
  - name: merge
    if:
      - rule: tautology
    then:
      - $merge("merge")
```

---

Please review all mock files to make sure we only have what's needed.